assignments/callbacks.js

```diff
@@ -2,29 +2,88 @@ const items = ['Pencil', 'Notebook', 'yo-yo', 'Gum'];
 
 function firstItem(arr, cb) {
   // firstItem passes the first item of the given array to the callback function.
+  for(let i = 0; i < arr.length; i++) {
+    if (arr[i] === arr[0]) {
```

Comment:
Since `i` is starting at 0 `arr[i]` will be equivalent to `arr[0]`. The first element in the array will always be at `arr[0]`. So there was no need for this for loop. You could just pass `arr[0]` to the callback.